definition/data/sheets/CaseEvent.json

```diff
@@ -48,5 +47,15 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "ReferredToMediation", "Name": "Mediation pending", "Description": "Mediation pending", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "stayed", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "ResetClaimSubmissionOperationIndicators", "Name": "Reset Claim Submission Ops Ind", "Description": "Reset Claim Submission Operation Indicators", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"}
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ResetClaimSubmissionOperationIndicators", "Name": "Reset Claim Submission Ops Ind", "Description": "Reset Claim Submission Operation Indicators", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "GenerateOrder", "Name": "Generate Order", "Description": "Generate Order", "DisplayOrder": 1, "PreConditionState(s)": "readyForDirections", "PostConditionState": "orderForJudgeReview", "SecurityClassification": "Public", "CallBackURLAboutToStartEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-start", "RetriesTimeoutAboutToStartEvent": "5,10,15"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ReviewOrder", "Name": "Review Order", "Description": "Review Order", "DisplayOrder": 1, "PreConditionState(s)": "orderForJudgeReview", "PostConditionState": "orderForLAReview", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ActionReviewComments", "Name": "Action review comments", "Description": "Action review comments", "DisplayOrder": 1, "PreConditionState(s)": "orderForLAReview", "PostConditionState": "orderForJudgeReview", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "DrawOrder", "Name": "Directions", "Description": "Directions", "DisplayOrder": 1, "PreConditionState(s)": "orderForJudgeReview;approved", "PostConditionState": "orderDrawn", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToStartEvent": "5,10,15", "CallBackURLSubmittedEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/submitted", "RetriesTimeoutURLSubmittedEvent": "5,10,15"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "SuccessfulMediation", "Name": "Mediation successful", "Description": "Mediation successful", "DisplayOrder": 1, "PreConditionState(s)": "referredMediation", "PostConditionState": "open", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "FailedMediation", "Name": "Mediation failed", "Description": "Mediation failed", "DisplayOrder": 1, "PreConditionState(s)": "referredMediation", "PostConditionState": "readyForDirections", "SecurityClassification": "Public"},
```

Comment:
Please change the post mediation state of FailedMediation to Open

Reply:
👍